modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -102,7 +102,7 @@ The following procedure describes how to delete SSH connection data of the insta
 {productname} server uses credentails collected from the instance to connect to the RMT server and to download the packages using reposync.
 These credentials are refreshed every 10 minutes by taskomatic using the defined SSH connection data. Connection to RMT server always uses the last known authentication credentials collected from the {payg} instance.
 
-Up-to date status of the {payg} instance credentials collect is shown in the column Status or on the instance details page.
+Up-to-date status of the {payg} instance credentials collect is shown in the column [literal]``Status`` or on the instance details page.
```

Comment:
Wondering whether we can delete the expression "up to date" here?

Reply:
"The latest status of the {payg} instance credentials collect is show in....